tools/build_script_generator/openocd.cfg.in

```diff
@@ -1,8 +1,8 @@
 %% for path in collector_values["path.openocd"] | sort
-add_script_search_dir {{ path }}
+add_script_search_dir {{ path | modm.windowsify(escape_level=1) }}
 %% endfor
 %% for file in collector_values["openocd.source"] | sort
-source [find {{ file }}]
+source [find {{ file | modm.windowsify(escape_level=1) }}]
```

Comment:
I think these probably also need to be windowsified.

Reply:
Okay, this _seems_ to work, but I can't say for 100%. I don't have an ST board with me at the moment, but when I try programming it doesn't choke on the config, just can't find the device. I'll squash this as well.